src/main/scala/is/hail/stats/ComputeRRM.scala

```diff
@@ -83,6 +83,21 @@ object ToNormalizedRowMatrix {
   }
 }
 
+object ToIndexedRowMatrix {
+  def apply(vds: VariantDataset, useDosage: Boolean, sampleMask: Array[Boolean], completeSampleIndex: Array[Int]): (Array[Variant], IndexedRowMatrix) = {
+    require(vds.wasSplit)
+    val n = vds.nSamples
+    val variants = vds.variants.collect()
+    val variantIdxBc = vds.sparkContext.broadcast(variants.index)
+    val indexedRows = 
+      if (!useDosage)
+        vds.rdd.map { case (v, (va, gs)) => IndexedRow(variantIdxBc.value(v), Vectors.dense(RegressionUtils.hardCalls(gs, n, sampleMask).toArray)) } // FIXME: handle sparse?
```

Comment:
Think we can manage this without broadcasting the variant index. Can't we just `zipWithIndex` and work from there? The variants are already in order as per `OrderedRDD` backing the VDS (at least I'm pretty sure that's always the case). Just seems unnecessary to broadcast what could be kind of a big thing if there are a ton of variants.